utils/array.js

```diff
@@ -153,3 +153,7 @@ export function uniq(ary) {
 
   return out;
 }
+
+export function getUniqueLabelKeys(aryResources) {
```

Comment:
This could be defined a little clearer

```
  return aryResources
    .map(res => Object.keys(res.metadata.labels))
    .flat()
    .filter((value, index, self) => self.indexOf(value) === index)
    .sort();
```

or to reduce some of the iterations it could be 

```
  const uniqueObj = aryResources.reduce((res, r) => {
    Object.keys(r.metadata.labels).forEach(l => (res[l] = true));

    return res;
  }, {});

  return Object.keys(uniqueObj).sort();
```

or you could do something with a `Set` but by the time that's converted to an array to sort i'm not sure it's worth it.